### DISCUSSION TITLE
Fresh Reinstall

### DISCUSSION THREAD

**Discussion by unmanagedtn** [devmfc/debian-on-amlogic#31] [General]
Is th fresh reinstall process the same aftwer writing the bootloader to the device (usb reset 7 seconds etc ) ? or is this a one and done deal? I am using this for devlopment work of something and I am a horrid programmer so I tend to break things badly.... 👎  trying to get better.... 👍

**Comment by devmfc** [comment]
Once you have setup multiboot, you won't need to do that again and you don't need to hold the reset button anymore. So you should just be able to boot via USB and reinstall. 

If you however reflashed the stock firmware you'll have to follow the original procedure.
